### DISCUSSION TITLE
Question: Can I use sharp character in path?

### DISCUSSION THREAD

**Discussion by Alceatraz** [vuejs/router#2283] [Help and Questions] [answered]
Hello everyone:

Here is what I want:

I can use name instead path jump to this URL `foobar/#setting`

```
// By button
<RouterLink :to="{ name: 'setting' }">Setting</RouterLink>
// Or 
router.push({ name: 'setting' })
```

But if I open `foobar/#setting` from browser directly, Vue route will dispaly home page ( the root `/` ). 

So question is: Can It ? And how ?

Thanks!

--- 

I'm pretty new to vue and vue-router but after hours searching and testing, Seem's like it's impossiable ?

**Comment by Deckluhm** [comment, marked as answer]
It works well on a freshly created project so you are probably doing something wrong.

Can you provide a reproduction?

Also, since you are new to Vue Router, I suggest to use the [HTML 5 Mode](https://router.vuejs.org/guide/essentials/history-mode.html#HTML5-Mode) instead of the [Hash Mode](https://router.vuejs.org/guide/essentials/history-mode.html#Hash-Mode) (unless you have a good reason to do otherwise). It's the recommended mode (even if that's not the 1st one listed) because it generates nicer URLs and is better for SEO. There is a small server side configuration to make it work but that's simple and well documented.
